resources/integration_test/tests/query.yaml

```diff
@@ -0,0 +1,293 @@
+tests:
+  - description: select_star_from_grades_ordered
+    db: integration_test
+    sql: SELECT * FROM grades ORDER BY _id
+    ordered: true
+    skip_reason: SQL-659
+
+  - description: select_star_from_grades_unordered
+    db: integration_test
+    sql: SELECT * FROM grades
+    expected_result:
+      - [0, 84.5, 303, 3]
+      - [1, 97.4, 10329, 3]
+      - [2, 89.3, 342, 3]
+      - [3, 91.9, 204323, 3]
+      - [4, 87.5, 303, 5]
+      - [5, 74.4, 10329, 5]
+      - [6, 80.1, 342, 5]
+      - [7, 83.3, 204323, 5]
+      - [8, 78.5, 200, 5]
+    row_count: 9
+    ordered: false
+    expected_sql_type: [INTEGER, DOUBLE, INTEGER, INTEGER]
+    expected_catalog_name: ['', '', '', '']
+    expected_column_class_name: [int, double, int, int]
+    expected_column_label: [_id, score, studentid, testid]
+    expected_column_display_size: [10, 15, 10, 10]
+    expected_precision: [10, 15, 10, 10]
+    expected_scale: [0, 15, 0, 0]
+    expected_schema_name: ['', '', '', '']
+    expected_is_auto_increment: [false, false, false, false]
+    expected_is_case_sensitive: [false, false, false, false]
+    expected_is_currency: [false, false, false, false]
+    expected_is_definitely_writable: [false, false, false, false]
+    expected_is_nullable: [columnNullable, columnNullable, columnNullable, columnNullable]
+    expected_is_read_only: [true, true, true, true]
+    expected_is_searchable: [true, true, true, true]
+    expected_is_signed: [true, true, true, true]
+    expected_is_writable: [false, false, false, false]
+
+  - description: select_star_from_class_ordered
```

Comment:
What makes this test case distinct from the first test? What is this case going to cover that the first test case does not?

Reply:
I removed the extra select query tests and some aggregate function tests.
My initial thinking was to write tests for general SQL features to validate the expected data was present and run queries a user might run initially.  After discussing offline with Matt, the direction is to have queries that test the different aspects of JDBC such as column types.